ui/src/ducks/app/solutions.js

```diff
@@ -175,8 +177,10 @@ export function* prepareEnvironment(action) {
   } else {
     yield put(
       addNotificationErrorAction({
-        title: intl.translate('prepare_environment'),
-        message: intl.translate('env_preparation_failed', { envName }),
+        title: intl.translate('add_solution_to_configMap', {
```

Comment:
Let's be consistent and keep translation keys always lowercase please :) so `configmap`, not `configMap` (here and everywhere else, of course)